src/cartodb.js

```diff
@@ -1,6 +1,5 @@
-var isLeafletAlreadyLoaded = !!window.L;
+window.L = require('leaflet');
```

Comment:
Shouldn't we remove this too?

Reply:
This is the only require used. It is used by the internal bundle if we remove this line the Builder do not include Leaflet in the window context.